modules/datadict/README.md

```diff
@@ -0,0 +1,43 @@
+# Data Dictionary
+
+## Purpose
+
+The data dictionary module is used for viewing and modifying the
+LORIS data dictionary.
+
+## Scope
+
+The displays (and depending on permissions) manages the data
```

Comment:
What displays? Is it the pages?

I would remove the parenthesis as this belong to permission section

Reply:
sorry for the delay responding.. this is done now